ui/UISystem.cpp

```diff
@@ -502,9 +502,7 @@ int ui_DoFrame(bool input) {
     ui_DoWindowFocus();              //	determine window with current input focus.
     res = ui_ProcessFocusedWindow(); //	process focused window
 
-    //		int64_t cur_time = timer_GetMSTime();
-    while ((timer_GetTime() - UI_input.cur_time) < UI_FRAMETIME) {
-    };
```

Comment:
I like the general idea, but old logic is sleep rest of UI_FRAMETIME with delta of current time and last timestamp of UI_input struct. I measured some timings, and `ui_DoFrame()` takes about 10 ms (on Debug), and with this we should sleep (50 - 10) ms. So we should measure, how many of 50 ms we should sleep each frame.